src/components/login-page/LoginPage.jsx

```diff
@@ -0,0 +1,199 @@
+                        }}
+                    >
+                        <Typography variant="h5" fontWeight="fontWeightBold">
+                            Power organizer
+                        </Typography>
+                        <Typography gutterBottom color={mySecondaryColor}>
+                            Unleash your productivity
+                        </Typography>
+                    </Box>
+
+                    <Box>
+                        <Typography fontWeight="fontWeightBold" sx={labelsStyle}>
+                            Username:
+                        </Typography>
+                        <TextField
+                            required
+                            disabled={isDisabled}
+                            id="outlined-required"
+                            name="username"
+                            label="Required"
+                            value={userInput.username}
+                            onChange={(event) => handleInput(event)}
+                            sx={inputsStyle}
+                            fullWidth
+                        />
+                    </Box>
+
+                    <Box borderBottom={1} borderColor={mySecondaryColor} marginBottom="48px">
+                        <Typography fontWeight="fontWeightBold" sx={labelsStyle}>
+                            Password:
+                        </Typography>
+                        <TextField
+                            required
+                            disabled={isDisabled}
+                            name="password"
+                            label="Required"
+                            type="password"
+                            value={userInput.password}
+                            onChange={(event) => handleInput(event)}
+                            sx={inputsStyle}
+                            fullWidth
+                        />
+                        <Typography gutterBottom color="primary" fontSize={14} sx={{ cursor: 'pointer' }}>
+                            Forgot your password?
+                        </Typography>
+                    </Box>
+                </CardContent>
+
+                <Stack direction="row" sx={{ display: 'flex', justifyContent: 'space-between' }}>
+                    <Button
+                        variant="outlined"
+                        disabled={isDisabled}
+                        sx={{ color: 'black', border: '1px solid black', boxShadow: 3 }}
+                        // onClick={() => navigate('/home-page')}
+                    >
+                        Sing Up
+                    </Button>
+                    <Button
+                        variant="contained"
+                        disabled={isDisabled}
+                        sx={{ bgcolor: 'black', color: 'white', boxShadow: 3 }}
+                        // onClick={() => navigate('/home-page')}
+                        onClick={handleSubmitEvent}
+                        // onSubmit={handleSubmitEvent}

```

Comment:
submit events are generally handled by \<form> elements, but we're not using one in this case